packages/dd-trace/src/plugins/util/user-provided-git.js

```diff
@@ -75,6 +110,8 @@ function getUserProviderGitMetadata () {
     [GIT_COMMIT_AUTHOR_EMAIL]: DD_GIT_COMMIT_AUTHOR_EMAIL,
     [GIT_COMMIT_AUTHOR_DATE]: DD_GIT_COMMIT_AUTHOR_DATE
   })
+  validateGitMetadata(metadata)
+  return metadata
```

Comment:
with the above suggestion, we could do 
```suggestion
  return removeInvalidGitMetadata(metadata)
```